pkgs/development/libraries/science/math/tiny-cuda-nn/default.nix

```
@@ -157,5 +157,7 @@ in
       license = licenses.bsd3;
       maintainers = with maintainers; [connorbaker];
       platforms = platforms.linux;
+      # g++: error: unrecognized command-line option '-mf16c'
+      broken = stdenv.hostPlatform.system == "aarch64-linux";
```

```suggestion
    broken = stdenv.isAarch64
```

It's already limited to linux

---

I actually thought about doing this but thought that my choice was a bit more robust.
I reverted :)